README.md

```diff
@@ -221,6 +221,28 @@ gem install shopify_cli
    shopify-cli help
    ```
 
+## GraphQL
+
+This library also supports Shopify's new [GraphQL API](https://help.shopify.com/api/reference)
+via a dependency on the [graphql-client](https://github.com/github/graphql-client) gem.
+The authentication process (steps 1-5 under [Getting Started](#getting-started))
+is identical. Once your session is activated, simply construct a new graphql
+client and use `parse` and `query` as defined by
+[graphql-client](https://github.com/github/graphql-client#defining-queries).
+
+```ruby
+client = ShopifyAPI::GraphQL.new
+
+SHOP_NAME_QUERY = client.parse <<-'GRAPHQL'
+  shop {
```

Comment:
@nneufeld i played this in rails console, and getting GraphQL::ParseError: Parse error on "shop" (IDENTIFIER) at [1, 1] for shop {name} but works with {shop {name}}

And also, I'm having trouble creating a simple rails model/controller method.
I'm getting GraphQL::Client::DynamicQueryError Expected definition to be assigned to a static constant
I tried changing case for variable but no luck. Please help.